mikrotik/resource_dhcp_server.go

```diff
@@ -0,0 +1,129 @@
+package mikrotik
+
+import (
+	"context"
+
+	"github.com/ddelnano/terraform-provider-mikrotik/client"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func resourceDhcpServer() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: createDhcpServer,
+		ReadContext:   readDhcpServer,
+		UpdateContext: updateDhcpServer,
+		DeleteContext: deleteDhcpServer,
+		Schema: map[string]*schema.Schema{
+			"id": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"add_arp": {
+				Type:     schema.TypeBool,
+				Optional: true,
```

Comment:
The mikrotik default for this is false. Is there a reason you decided to stick to true? I'm not sure I understand the use case of this setting from the mikrotik docs, but on my network this is disabled.

Reply:
Yep, you are correct 🤦